tracer/span_test.go

```diff
@@ -35,13 +35,13 @@ func TestSpanSetMeta(t *testing.T) {
 
 	// check the map is properly initialized
 	span.SetMeta("status.code", "200")
-	assert.Equal(len(span.Meta), 1)
 	assert.Equal(span.Meta["status.code"], "200")
 
 	// operating on a finished span is a no-op
+	nMeta := len(span.Meta)
 	span.Finish()
 	span.SetMeta("finished.test", "true")
-	assert.Equal(len(span.Meta), 1)
+	assert.Equal(len(span.Meta), nMeta)
```

Comment:
Mea culpa, I had noticed but I was lazy ;) It's fixed now.
For the record :%s/\(assert[^(]*\)(\(.*\), \(.*\))/\1(\3, \2)/gc in vim does the work pretty well.